demos/SanicJymper/main.cpp

```diff
@@ -32,7 +154,14 @@ int main(int argc, char* argv[])
 
     if (unicornEngine->Init())
     {
-        unicorn::video::Graphics* pGraphics = unicornEngine->GetGraphics();
+        pGraphics = unicornEngine->GetGraphics();
+
+        const std::map<uint32_t, unicorn::system::input::Gamepad*>& gamepads = unicornEngine->GetGamepads();
```

Comment:
I'm really not sold on the auto idea - to learn what the type is developer would have to open the header?